.github/workflows/firebase.yml

```diff
@@ -68,4 +70,4 @@ jobs:
           gcloud firebase test android run --type instrumentation --use-orchestrator \
           --environment-variables clearPackageData=true \
           --app app-debug/app-debug.apk --test app-debug-androidTest/app-debug-androidTest.apk \
-          --device model=blueline,version=28,locale=en,orientation=portrait --num-flaky-test-attempts 3
\ No newline at end of file
+          --device model=flame,version=29,locale=en,orientation=portrait --num-flaky-test-attempts 3
```

Comment:
Please explain why you increased version here

Reply:
Because we have more phones with API 29 vs API 28 and comparing to API 30 there are no disabled tests. Should I add it into a comment?